src/cli.rs

```diff
@@ -312,6 +304,15 @@ pub enum EdgeAppVersionCommands {
         #[arg(short, long, action = clap::ArgAction::SetTrue)]
         json: Option<bool>,
     },
+    Promote {
+        /// Edge app revision to promote.
+        revision: u32,
```

Comment:
revision shopuld be named argument, so please add short, long.
There will be no positional arguments